companies/services/kis_quote.py

```
@@ -97,6 +101,70 @@ def _get_access_token(self) -> Optional[str]:
+            lock_acquired = cache.add(
+                KIS_RATE_LIMIT_LOCK_KEY, "locked", timeout=lock_ttl
+            )
+            if lock_acquired:
+                break
+
+            # 락 획득 실패 시 짧은 backoff 후 재시도
+            if attempt < max_retries - 1:
+                time.sleep(retry_backoff)
+            else:
+                logger.warning(
+                    f"KIS rate limit 락 획득 실패 (최대 재시도 횟수 초과). "
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)